README.md

```diff
@@ -26,6 +26,16 @@ Or install it yourself as:
 
 $ gem install waistband
 
+## Compatibility
+Waistband relies on [elasticsearch-ruby](https://github.com/elastic/elasticsearch-ruby), which
+mirrors its versioning with Elasticsearch.
+
+    | Gem Version   |   | Elasticsearch |
+    |:-------------:|:-:| :-----------: |
+    | 0.x           | → | 1.x           |
+    | 6.x           | → | 6.x           |
+    | master        | → | 6.x           |
+
```

Comment:
Here's the full compatibility of `elasticsearch-ruby`:

  | Gem Version   |   | Elasticsearch |
  |:-------------:|:-:| :-----------: |
  | 0.90          | → | 0.90          |
  | 1.x           | → | 1.x           |
  | 2.x           | → | 2.x           |
  | 5.x           | → | 5.x           |
  | 6.x           | → | 6.x           |
  | master        | → | master        |

If we want, we could easily add support for other versions (except for Elasticsearch 0.9 as that'd require breaking 0.x of Waistband)

Reply:
@bleonard do you have any issues with mirroring this sort of versioning?